Cubical/Categories/Instances/CommRings.agda

```diff
@@ -105,6 +105,24 @@ module _ {ℓ : Level} where
   snd (fst (snd TerminalCommRing y)) = makeIsRingHom refl (λ _ _ → refl) (λ _ _ → refl)
   snd (snd TerminalCommRing y) f = RingHom≡ (funExt (λ _ → refl))
 
+  open RingTheory
+  trivialIsTerminalCommRing : ∀ (R : CommRing ℓ)
```

Comment:
This will not be needed once we merged https://github.com/agda/cubical/pull/1102. Is it still worth keeping this little result?

Reply:
@MatthiasHu and I decided to delete it. We agreed that it would be better to have a file where all the different ways of being a trivial ring (being contractible, 0=1, being terminal etc.) are introduced and proven equivalent.